config.py

```diff
@@ -158,25 +182,25 @@ def getenv_unique_set(var_name, default=frozenset()) -> frozenset[str]:
     # private channel is where stampy logging gets printed
     bot_private_channel_id = getenv("BOT_PRIVATE_CHANNEL_ID", default=None)
     # NOTE: Rob's invite/member management functions, not ported yet
-    can_invite_role_id = getenv_unique_set("CAN_INVITE_ROLE_ID", default=None)
     member_role_id = getenv("MEMBER_ROLE_ID", default=None)
     # bot_reboot is how stampy reboots himself
-    bot_reboot = getenv("BOT_REBOOT", default=False)
+    bot_reboot = getenv("BOT_REBOOT", default=None) is not None
```

Comment:
This is not a Boolean, please undo

Reply:
Sorry, undone. Although... is there a reason not to make it a `bool` if there are exactly two values allowed for this variable? (`"exec"` and `False`).